### ISSUE TITLE
Update android publisher dependency to v3

### ISSUE THREAD

**Issue by agap** [bitrise-steplib/steps-google-play-deploy#49]
Starting from Dec. 1 2019 versions 1 and 2 of Google Play Developer API will [no longer be available](https://android-developers.googleblog.com/2019/03/changes-to-google-play-developer-api.html), hence it's required to switch to v3 before that date in order to keep this step working.

**Comment by fehersanyi-bitrise** [comment]
Hello, thank You for the heads up, I created a card for the tooling team, to investigate the issue and update the dependency as needed.

**Comment by marianeum** [comment]
Hi @fehersanyi-bitrise - do you have any update on this?

**Comment by bitce** [comment]
Hi @marianeum!

This isn't something we're actively looking into at the moment but the planning is laid out, don't worry, this change will happen before December and until then please ignore the warnings.

**Comment by Frank1234** [comment]
Would be great when this is fixed soon, so we can be sure that this is the only third party causing this message (and our apps are not using API 1 or 2 in any other way).

**Comment by bitce** [comment]
Hey @Frank1234! The team has been looking at this since the announcement, don't exactly treat is as an issue that needs fixing, it's more of a deadline we have to attend to. Due to different issue in the step we already had to work with the newer version of the distribution API, so it might happen sooner than we originally thought 🙂

In any event, we can guarantee that this will not cause any other inconveniences besides the unwanted warning messages.

**Comment by rjstelling** [comment]
It would be nice to be able to test this before the 1st December!?!

Is there a branch with the fix we can test? I'm happy to help with beta testing! ❤️

**Comment by cookseym** [comment]
** bump ** 👂

**Comment by prcaen** [comment]
Hello,
We are not so far from the deadline and it would be really nice to have it really soon.
We will not be able to deploy on the Play Store thanks Bitrise, and lot of customers will complain...
Thanks

**Comment by Guuz** [comment]
I'm also a little bit scared with the deadline coming closer... Google makes it sound like a hard deadline and taking the API offline on December 1st...

**Comment by tdekoning** [comment]
We are also using this step and would like to see that v3 is supported before the deadline expires.
Any progress update on this issue?

**Comment by tdekoning** [comment]
Seems like #63 should fix this?

**Comment by williamsandowaracle** [comment]
Hi there, we are waiting on this as well. It would be handy to have this merged as soon as possible so we can be sure we aren't using any other 3rd party libraries using the deprecated API.

**Comment by lpusok** [comment]
Hi all,
The new release is now available https://github.com/bitrise-steplib/steps-google-play-deploy/releases/tag/3.0.0. In case it does not appear instantly in the workflow editor cleaning browser caches may be needed.

@williamsandowaracle @tdekoning @Guuz @agap @marianeum @prcaen @cookseym @rjstelling @Frank1234 and everyone, thanks for your patience, please report if you encounter any issues with the new release.

**Comment by agap** [comment]
Nice, thanks for the update!

**Comment by prcaen** [comment]
Thanks

**Comment by lpusok** [comment]
Closing as the new release is available.
